### PR TITLE
BUG fix：日志会同时输出到两个地方

### PR DESCRIPTION
**Pull request by CosmosSun**
https://github.com/iqiyi/qnsm/issues/9
解决issue

### CODE REVIEW

File: include/qnsm_cfg.h [hunk 1]
```diff
@@ -251,8 +251,6 @@ int32_t qnsm_match_all_net_segment(enum en_qnsm_ip_af af, QNSM_IN_ADDR *in_addr)
 */
 inline int32_t qnsm_match_service(void *data, uint16_t port);
 void *qnsm_get_kafka_cfg(const char *name);
-void qnsm_log_init(void);
-
```
Comment (mflu) [inline include/qnsm_cfg.h]: why do u delete this line?

File: src/qnsm_cfg.c [hunk 2]
```diff
@@ -896,7 +896,7 @@ redisContext* qnsm_cfg_init_redis_ctx(void)
 
             /*auth*/
             while(NULL == (reply = redisCommand(c, cmd)));
-            QNSM_LOG(INFO, "%llu redis connect success\n", jiffies());
```
Comment (mflu) [inline src/qnsm_cfg.c]: why change this?

File: include/qnsm_cfg.h (hunk 1 above)
Comment (CosmosSun) [reply include/qnsm_cfg.h]: Deprecated code, now replaced with static func qnsm_init_log

File: src/qnsm_cfg.c (hunk 2 above)
Comment (CosmosSun) [reply src/qnsm_cfg.c]: > 
> 
> why change this?

QNSM_LOG default format include timestamp，so remove this duplicate code

**Pull request merged**